.github/workflows/bluetooth-tests.yaml

```diff
@@ -41,7 +41,13 @@ jobs:
         uses: actions/checkout@v2
 
       - name: west setup
+        env:
+          BASE_REF: ${{ github.base_ref }}
         run: |
+          git config --global user.email "you@example.com"
+          git config --global user.name "Your Name"
+          git remote -v
```

Comment:
Are these the name and email we want?

Reply:
this is just so that Git doesn't complain, it won't be used at all.

Reply:
I see that some of the other Github actions use 
```
          git config --global user.email "actions@zephyrproject.org"
          git config --global user.name "Github Actions"
          git remote -v
```

Reply:
There's also one using 
```
          git config --global user.email "bot@zephyrproject.org"
          git config --global user.name "Zephyr Bot"
```

Reply:
Both of these alternatives seem cleaner to me.

Reply:
As I mentioned, we rebase here, this user config will never be actually used.